charts/newrelic-infrastructure/templates/controlplane/daemonset.yaml

```diff
@@ -133,10 +133,9 @@ spec:
                   fieldPath: "spec.nodeName"
             {{- end }}
             {{- end }}
-            {{- if (include "newrelic.customAttributes" .) }}
+            {{- with fromYaml ( include "common.customAttributes" . ) }}
             - name: "NRIA_CUSTOM_ATTRIBUTES"
-              value: >-
-                {{- include "newrelic.customAttributes" . | nindent 16 }}
+              value: {{ . | toJson | quote }}
```

Comment:
It is changed a commit later because this is to make progressive changes.

`customAttributes` are handled by the agent config and is removed here: https://github.com/newrelic/nri-kubernetes/pull/383/commits/1c85539e2722fd33671d5fae4becb42fbfbc2b8d#diff-d16718b05aef34f809eb79603c6d3f59c281721e380661893e9b6c95c525e27aL136-L151

I am resolving this thread :)